src/main/java/com/theironyard/Entities/Entry.java

```diff
@@ -0,0 +1,91 @@
+package com.theironyard.Entities;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+/**
+ * Created by darionmoore on 1/11/17.
+ */
+@Entity
+@Table(name = "entries")
+public class Entry {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column(nullable = false)
+    private String entryTitle;
```

Comment:
Do not name variables inside of a class with the name of the class.  `entryTitle` is redundant since it is already in the entry.  It looks ridiculous in code `entry.entryTitle`